test/utils_test/enumerable_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule RecaseEnumerableTest do
+  use ExUnit.Case
+
+  test "should convert keys of a map" do
+    assert Recase.Enumerable.convert_keys(
+             %{"upper case" => "value", "upper-case2" => "value"},
+             &Recase.to_pascal/1
+           ) == %{"UpperCase" => "value", "UpperCase2" => "value"}
+  end
+
+  test "should convert keys of a nested map" do
+    assert Recase.Enumerable.convert_keys(
+             %{"upper case" => %{"upper-case2" => "value"}},
+             &Recase.to_pascal/1
+           ) == %{"UpperCase" => %{"UpperCase2" => "value"}}
+  end
+
+  test "should convert keys of a map in list" do
+    assert Recase.Enumerable.convert_keys(
+             [%{"upper case" => "value", "upper-case2" => "value"}],
+             &Recase.to_pascal/1
+           ) == [%{"UpperCase" => "value", "UpperCase2" => "value"}]
+  end
+
+  test "should convert keys of a nested map in list" do
+    assert Recase.Enumerable.convert_keys(
+             [%{"upper case" => %{"upper-case2" => "value"}}],
+             &Recase.to_pascal/1
+           ) == [%{"UpperCase" => %{"UpperCase2" => "value"}}]
+  end
+
+  test "should return the same list when list items are usual text" do
+    assert Recase.Enumerable.convert_keys(
+             ["upper case", "upper-case2"],
+             &Recase.to_pascal/1
```

Comment:
Can you, please, also write a test with something like:

```elixir
Recase.Enumerable.convert_keys(..., &Recase.to_path(&1, "\\"))
```

So we can be sure that two arguments are fine as well.